livecheck/main.py

```diff
@@ -302,7 +307,7 @@ def do_main(*, auto_update: bool, cat: str, ebuild_version: str, parsed_uri: Par
             else:
                 raise UnhandledState(cat, pkg, url)
         else:
-            raise UnhandledState(cat, pkg, url)
+            results = [version]
```

Comment:
What is the thought process behind this change?

Reply:
Yes, at this point I looked at how passing the pecl version returns the latest version available, but when performing the yield I did not find another possibility to pass that information, I saw how it passed the data 'download.jetbrains.com' and I saw that only null was passed in that possibility, so use in line 246 

<pre>
yield (cat, pkg, ebuild_version, new_version, url, None, True)
</pre>